install_config/install/deploy_router.adoc

```diff
@@ -230,6 +230,45 @@ endif::[]
 You can link:../../admin_guide/high_availability.html[set up a highly-available
 router] on your OpenShift cluster using IP failover.
 
+
+[[configuring-router-sharding]]
+=== Configuring Router Sharding
+
+Router sharding is
+link:../../architecture/core_concepts/routes.html#router-sharding[based on labels];
+you set labels on the router
+and make sure there is a corresponding node selector
```

Comment:
Might be a good chance to link to the node selector docs:

https://docs.openshift.com/enterprise/3.1/admin_guide/managing_projects.html#using-node-selectors